lib/biokbase/catalog/Impl.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, config):
         #BEGIN_CONSTRUCTOR
         print('Starting the Catalog service.  Service configuration:')
         for c in config:
-            if c == 'nms-admin-psswd':
+            if c == 'nms-admin-psswd' or c == 'nms-admin-token':
```

Comment:
If you want you could just strip out the user/pwd config items. They're obsolete.

Reply:
Done